tasks/firewalld.yml

```
@@ -22,4 +22,11 @@
     name: "{{ __firewall_packages_base }}"
     state: present
     use: "{{ (__firewall_is_ostree | d(false)) |
-             ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
+      ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
+  register: firewall_package_result
+
+- name: Reboot transactional update systems
+  reboot:
+  when:
+    - firewall_package_result.changed
```

```suggestion
    - firewall_package_result is changed
```
the `is changed` style is recommended now - https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tests.html#testing-task-results